plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/SimpleSortTests.java

```diff
@@ -450,7 +417,7 @@ public void testScoreSortDirection_withFunctionScore() throws Exception {
         assertThat(searchResponse.getHits().getAt(2).getId(), equalTo("3"));
 
         searchResponse = client().prepareSearch("test")
-                .setQuery(functionScoreQuery(matchAllQuery(), scriptFunction(new Script("_source.field"))))
+                .setQuery(functionScoreQuery(matchAllQuery(), fieldValueFactorFunction("field")))
```

Comment:
Should we have a tests for these inline scripts as well as fieldValueFactorFunctions? Like a small one in the groovy plugin?